actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*.  Raises `ActionController::RespondToMismatchError` with confliciting `respond_to` invocations.
+
+    `respond_to` can match multiple types and lead to undefined behavior when
```

Comment:
Let me know if I'm not using language that isn't idiomatic with `respond_to` (or any other changes you would like).